mkdocs-base.yml

```diff
@@ -147,6 +147,7 @@ nav:
   - Welcome to Percona Everest: index.md 
   - Release Notes:
       - Release notes index: release-notes/release_notes_index.md
+      - Percona-Everest 1.2.0 (2024-09-09): release-notes/Percona-Everest-1.2.0-(2024-09-09).md
```

Comment:
As usual commenting so that we don't forget to update the date :slightly_smiling_face: